testing/scenario_app/ios/Flutter/Generated.xcconfig

```diff
@@ -0,0 +1,7 @@
+// This is a generated file; do not edit or check into version control.
```

Comment:
I think you didn't mean to check this in

Reply:
done